pytest_flask/fixtures.py

```diff
@@ -34,35 +36,38 @@ def login(self, email, password):
                 return self.client.post(url_for('login'), data=credentials)
 
             def test_login(self):
-                assert self.login('vital@example.com', 'pass').status_code == 200
+                assert self.login('vital@foo.com', 'pass').status_code == 200
```

Comment:
I'll change it back - I was just trying to get the line to squeeze into 80 chars so my pep8 checker would stop complaining :)